hack/verify-chart-version.sh

```diff
@@ -18,13 +18,14 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-if [ -z "${1:-}" ]; then
+chart_tarball=${1:-}
+DOCKER=${DOCKER:-docker}
+
+if [ -z "${chart_tarball}" ]; then
```

Comment:
note: I confirmed that this works locally with rootless podman instead of docker.

(I don't have docker installed ATM)